### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by dieum512**
In this branch I:

- Created the HTML, CSS, and JS files
- Divided the JS file into some modules
- Updated the README file and 
- Checked for and corrected some linter errors

### CODE REVIEW

File: UI.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Store from './store.js';
+// UI for UI tasks
+export default class UI {
+  static displayBooks() {
```
Comment (muneebulrehman) [inline UI.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: UI.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBook(book));
+  }
+
+  static addBook(book) {
```
Comment (muneebulrehman) [inline UI.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: UI.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    row.innerHTML = `
+                  <td>${book.title}</td>
+                  <td>${book.author}</td>
+                  <td><a href="#" class="remove-btn">Remove</a></td>
+              `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
```
Comment (muneebulrehman) [inline UI.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: UI.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
```
Comment (muneebulrehman) [inline UI.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: store.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+// handle storage
+export default class Store {
+  static getBooks() {
```
Comment (muneebulrehman) [inline store.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: store.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
```
Comment (muneebulrehman) [inline store.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

File: store.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removBook(title) {
```
Comment (muneebulrehman) [inline store.js]: kindly use the arrow functions as per the requirements of the project. ⬇️
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @dieum512 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good docs

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly rename your `script.js` file to `index.js`. It is one of the project requirements and also a convention in the JS world. The main file is generally named as `index.js`.⬇️
image


- [ ] Kindly move all the modules you have created to the `modules` directory. It is one of the project requirements. This will help you keep the project clean and you know what files are where. ⬇️
image

- [ ] You need to display the current date below the navigation bar, and for this, you need to use the Luxon library. Kindly implement this feature and then submit for a re-review.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/UI.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+                  <td>by${book.author}</td>
+                  <td><a href="#" class="remove-btn">Remove</a></td>
+              `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook = (el) => {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.parentElement.remove();
```
Comment (KayLemba) [inline modules/UI.js:26]: As suggested by the previous code reviewer  You need to display the current date below the navigation bar, and for this, you need to use the Luxon library. Kindly go through project requirements and implement this feature.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+  align-items: center;
+}
+
+.form-container h1 {
+  color: #08410c;
+  text-align: center;
+}
+
+.book-form {
+  /* margin-top: 1000px; */
```
Comment (KayLemba) [inline style.css]: Please kindly get rid of unused code for it makes your code base look untidy.

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @dieum512    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @dieum512,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**